### ISSUE TITLE
Show the range from begin month

### ISSUE THREAD

**Issue by eleave** [damienroche/vue-mj-daterangepicker#24]
Hi, Damien!
You made a great date range picker! 
I have a question about start and finish range date begin.

**What's the point?**
Today is 09 March. I set the date range from 15 March to 05 April and save it.
Then I refresh the page, after page is loaded I see that current month setted as April (because the end range is next month and i do not see March), it would be better if calendar could begin from Start range - March!

**What I see when page is loaded:** (End of the range)
![image](https://user-images.githubusercontent.com/34356472/76249092-aa77b180-6253-11ea-8170-26d2c3d90092.png)

**What i expected to see:** (Start of the range)
![image](https://user-images.githubusercontent.com/34356472/76249467-6e911c00-6254-11ea-8bb0-db444061a9bc.png)

I try to use **begin** param with a start date, but it does not help.

```
<date-range-picker 
    ref="rangepicker"
    :begin="'{{ $item->start }}'"
    :from="'{{ $item->start }}'" 
    :to="'{{ $item->finish }}'" 
    :past="false" 
/>
```

Btw, it would be useful to add a Home icon (or a method to quickly back to the current day, but without it selected)

**Comment by damienroche** [comment]
Hi @eleave thanks for you feedback i will add a `rangeDisplayed` Prop so you can select which range to display between `to` and `from`

**Comment by damienroche** [comment]
available now in v0.1.11
